tests/integration/true_positive_test.py

```diff
@@ -125,27 +143,38 @@ def testOnlyScoreFirstTruePositiveWithinWindow(self):
 
     timestamps = generateTimestamps(start, increment, length)
     windows = generateWindows(timestamps, numWindows, windowSize)
-    labels = generateLabels(timestamps, windows)
```

Comment:
You can remove `generateLabels` from the test helpers now.

Reply:
Helper removed and tests fixed.